api/provision/csp-write-portfolio.test.ts

```diff
@@ -17,12 +15,11 @@ import {
 // Reused mocks
 jest.mock("./csp-configuration");
 const mockedConfig = cspConfig.getConfiguration as jest.MockedFn<typeof cspConfig.getConfiguration>;
-jest.mock("axios");
-const mockedAxios = axios as jest.Mocked<typeof axios>;
 jest.mock("../../idp/client");
 const mockedGetToken = idpClient.getToken as jest.MockedFn<typeof idpClient.getToken>;
 
-describe("Successful invocation of mock CSP function", () => {
+// Skipped because this covers old mock behavior
+describe.skip("Successful invocation of mock CSP function", () => {
```

Comment:
Should we just remove old mock behavior instead of skipping it?

Reply:
Yeah, the same thought crossed my mind. I thought they were more of a placeholder for new tests.